scripts/configure-venue-access.ts

```diff
@@ -0,0 +1,141 @@
+import fs from "fs";
+import admin from "firebase-admin";
+import serviceAccount from "./sparkle-7-firebase-adminsdk-f5a3m-482bebb057.json";
+import "firebase/firestore";
+import { Venue } from "../src/types/Venue";
+import {
+  VenueAccessCodes,
+  VenueAccessEmails,
+  VenueAccessType,
+} from "../src/types/VenueAcccess";
+
+const mergeStringArrays = (
+  arr1: string[] = [],
+  arr2: string[] = []
+): string[] => {
+  const result = (arr1 ?? []).map((val) => val.trim());
+  (arr2 ?? []).forEach((val) => {
+    const valTrimmed = val.trim();
+    if (!result.includes(valTrimmed)) {
+      result.push(valTrimmed);
+    }
+  });
+  return result;
+};
```

Comment:
It looks like the intent of this code is to create an array that contains the elements of `arr1` and `arr2`, without any duplicates? If so, we can just use a `Set` or similar for this:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

If we do end up needing this helper still, it should probably be moved into `scripts/lib/helpers`